bootstrap.php

```diff
@@ -14,4 +14,19 @@
  *
  */
 
-require_once __DIR__ . '/vendor/autoload.php';
+$vendorDir = __DIR__ . '/vendor';
+$importedVendorDir = __DIR__ . '/../../../vendor';
+
+/**
+ * Check whether this SDK got imported via composer. Composer imports required libraries to
+ * vendor/<vendor_namespace>/<project_namespace>
+ */
+if (is_dir($importedVendorDir)) {
+    $vendorDir = $importedVendorDir;
+    error_log(
+        'Warning: You are using composer to import the Learnosity SDK.' .
```

Comment:
I think the error should say that the `bootstrap.php` is only valid when installed from a dist zip, but this is not the case here (as our `$vendorDir` is missing), and that we are now trying the composer autoloader (as a convenience feature).